### ISSUE TITLE
The FIRAnalytics(Consent) category is not part of the umbrella header in 7.2.0

### ISSUE THREAD

**Issue by kaybutter** [firebase/firebase-ios-sdk#7114]
### [REQUIRED] Step 1: Describe your environment

  * Xcode version: 12.2
  * Firebase SDK version: 7.2.0
  * Installation method: `Swift Package Manager`
  * Firebase Component: Analytics

### [REQUIRED] Step 2: Describe the problem

I'm trying to set the consent mode in the latest FirebaseAnalytics using the `setConsent(_:)` method as documented [here](https://firebase.google.com/docs/reference/swift/firebaseanalytics/api/reference/Categories/FIRAnalytics(Consent)). Xcode complains, that the method cannot be found. It worked in 7.1.0.

Looking inside the FirebaseAnalytics.xcframework, I can see that the header file declaring the method is there. But it is not included in the umbrella header. Because of that it cannot be used from swift, by just importing `FirebaseAnalytics`.

#### Steps to reproduce:

1. Integrate FirebaseAnalytics 7.2.0
2. Try to use the `setConsent` method.
3. Try to build your app

Xcode will now complain that:

> Type 'Analytics' has no member 'setConsent'

#### Relevant Code:

``` swift
import FirebaseAnalytics

...

Analytics.setConsent([.adStorage: .denied, .analyticsStorage: .denied])
```

**Comment by ryanwilson** [comment]
Sorry for the issues and thanks for the report. Our ZipBuilder tool uses CocoaPods under the hood and it looks like the bug with a missing `-ObjC` flag may be hit, causing this. I'll see if we can publish a new patch with the fix soon.

In the meantime, @MapaX discovered a workaround: if you add the `#import "FIRAnalytics+Consent.h"` to `Headers/FirebaseAnalytics.h` it should work for now. (from https://github.com/firebase/firebase-ios-sdk/issues/7117#issue-758457404)

**Comment by paulb777** [comment]
The recently added `FIRAnalytics+Consent.h` is missing from the `FirebaseAnalytics.h` umbrella headers.

Googlers, see the fix at cl/345499702

**Comment by paulb777** [comment]
Fixed in today's 7.3.0 release.
